### ISSUE TITLE
Force a fixed number-of-days, even with hiddenDays

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#2753]
Originally reported on Google Code with ID 2488

```
I created a custom 3 day view calendar without displaying sundays (using hiddenDays[0]).

The problem is when I use prev/next buttons, sometimes the calendar displays 3 days,
sometimes only 2.
This problem occurs when there's a sunday in the 3 days to display.


To reproduce :
Go to http://jsbin.com/qavazakame/1/

Click next button, when there's a sunday in the 3 days, the calendar displays only
2 days.

For example, when calendar loads, it displays Wed 11/12 as first day. 
When clicking next button, it displays only 2 days : Saturday and Monday instead of
displaying 3 days.


```

Reported by `fbouvier39` on 2015-04-21 14:19:20

Imported with **1** stars.

**Comment by arshaw** [comment]
i understand the need for this, but im hesitant to adjust the current behavior, which is defined through a custom view's `duration`.

`duration` is meant to represent the duration from the start date through until the end date. this might be misleading.

another setting called `{ days: 3 }` might be more appropriate

**Comment by vlapo** [comment]
+1

**Comment by espen** [comment]
@vlapo +1 does nothing but trigger an email to people subscribed to this issue. Please see https://fullcalendar.io/wiki/Requesting-Features/ regarding making your vote count.

**Comment by arshaw** [comment]
Released in **v3.3.0**
Blog post: https://fullcalendar.io/blog/2017/03/fullcalendar-33-and-scheduler-16/

Use [dayCount](https://fullcalendar.io/docs/current_date/dayCount/) instead of `duration`.
